vtex-apps/google-tag-manager
# Feat/ga4 new login event

#### What is the purpose of this pull request?

This PR intends to add the `login` GA4 event to be sent when `vtex:login` is received.

[`login` event](https://developers.google.com/analytics/devguides/collection/ga4/reference/events?hl=en&client_type=gtag#login)|
-|
<img width="940" alt="Captura de Tela 2023-03-14 às 18 17 51" src="https://user-images.githubusercontent.com/36740164/225138439-0dc0fac0-f3f0-4b0d-a0dc-25e12747c0bd.png">




#### How should this be manually tested?

- In your local environment, inside the `google-tag-manager` app repository, go to the branch `feat/ga4-new-login-event`;
- Login in your preferred account and workspace;
- Link the app;
- Go to the Admin and access the App Store. Look for the Google Tag Manager app and access the `Settings`;
- Check the `Merge Universal Analytics and Google Analytics 4 Events` option;
- Access the store and perform a login with any method;
- Check the `dataLayer` object (just type `dataLayer` in the console and press enter). The event data should be in the list.

#### Types of changes
